server/cmd/dcrdex/config.go

```diff
@@ -55,6 +55,7 @@ const (
 	defaultRegFeeConfirms   = 4
 	defaultRegFeeAmount     = 1e8
 	defaultBroadcastTimeout = 5 * time.Minute
+	defaultMiaUserTimeout   = 90 * time.Second
```

Comment:
A bit short imo.

Reply:
Might be.  60 seconds was tossed around, and I bumped it to 90.  Since it's a `--miatimeout` switch, I think we can leave it as is for now.

Reply:
I'm with @JoeGruffins. In fact, I think we should just set this to the broadcast timeout and forget the operator variable.

Reply:
Done.  Now set from broadcast timeout setting.